pesacheck_meedan_bridge/.env.example

```diff
@@ -0,0 +1,6 @@
+PESACHECK_URL=https://pesacheck.org/feed
+PESACHECK_RSS_2_JSON_API_KEY=
+PESACHECK_CHECK_URL=https://check-api.checkmedia.org/api/graphql
+PESACHECK_TOKEN=
+PESACHECK_WORKSPACE_SLUG=pesacheck-tipline-sandbox
```

Comment:
What is this part of? If Check, name it accordingly e.g. `PESACHECK_CHECK_WORKSPACE_SLUG`